testsuites/CBLTester/p2p_tests/test_p2p_replication_filtering.py

```diff
@@ -71,7 +71,7 @@ def test_p2p_replication_push_pull_filtering(params_from_base_test_setup, server
     # Now set up client
     repl_config = p2p_client.configure(port=url_listener_port, host=server_host, server_db_name=db_name_server,
                                        client_database=cbl_db_client, continuous=False,
-                                       replication_type="push_pull", endPointType=endpoint_type)
+                                       replication_type=replicator_type, endPointType=endpoint_type)
```

Comment:
I think here is preparing the env and data, don't need to use replicator_type parameter. for instance, if replicator_type is pull, the docs created locally will never be able to be replicated to remote, the subsequence test logic will need to be changed.